ci_build_images/opensuse15.Dockerfile

```diff
@@ -0,0 +1,50 @@
+#
+# Buildbot worker for building MariaDB
+#
+# Provides a base OpenSUSE image with latest buildbot worker installed
+# and MariaDB build dependencies
+
+ARG base_image
+FROM "$base_image"
+LABEL maintainer="MariaDB Buildbot maintainers"
+
+# Install updates and required packages
+RUN zypper update -y && \
+    zypper install -y -t pattern devel_basis && \
+    zypper install -y git ccache subversion \
+    python-devel libffi-devel openssl-devel glibc-locale\
+    python-pip curl wget && \
+    # install MariaDB dependencies
+    #zypper mr -er repo-source
+    zypper -n si -d mariadb
+
+# Create buildbot user
+RUN useradd -ms /bin/bash buildbot && \
+    mkdir /buildbot && \
+    chown -R buildbot /buildbot && \
+    curl -o /buildbot/buildbot.tac https://raw.githubusercontent.com/MariaDB/mariadb.org-tools/master/buildbot.mariadb.org/dockerfiles/buildbot.tac
+WORKDIR /buildbot
+
+# upgrade pip and install buildbot
+RUN pip install -U pip virtualenv && \
+    pip install --upgrade setuptools && \
+    pip install buildbot-worker && \
+    pip --no-cache-dir install 'twisted[tls]'
```

Comment:
Ideally we should skip this step since pip is very error prone for future builds. Please check if the buildbot-worker package is available via the package manager similar to other OSs (https://github.com/MariaDB/buildbot/blob/de9660be1e0bbfe608f23a27560ae4d0d9857431/ci_build_images/fedora.Dockerfile#L16). If that is not available we can further discuss this aspect.